about.html

```diff
@@ -1,511 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Hey @kaka5277, just double checking. Why were the `about.html` and `index.html` pages deleted?

Once these deletions are checked in, the files will disappear for everyone.